pkg/api/v1/project_settings.go

```diff
@@ -9,6 +9,7 @@ import (
 type ProjectSettings struct {
 	IsCollectDatabaseSpecificsStatisticsEnabled *bool `json:"isCollectDatabaseSpecificsStatisticsEnabled,omitempty"`
 	IsDataExplorerEnabled                       *bool `json:"isDataExplorerEnabled,omitempty"`
+	IsExtendedStorageSizesEnabled               *bool `json:"isExtendedStorageSizesEnabled,omitempty"`
```

Comment:
Please make sure we have a ticket for DOCS team to reflect this change

Reply:
Done